securetea/core.py

```diff
@@ -110,7 +133,13 @@ def on_move(self, x, y):
         self.twitter.notify(msg)
 
         # Send a warning message via telegram bot
-        self.telegram.notify(msg)
+        try:
+            self.telegram.notify(msg)
+        except:
+            self.logger.log(
+                "Telegram credentials not configured. The application will continue to run", 
+                logtype="warning"
```

Comment:
Provide the message only if the configuration failed. As we could not force any integration